### ISSUE TITLE
Closing parenthesis is converted to closing curly bracket during export

### ISSUE THREAD

**Issue by agov** [mongo-express/mongo-express#234]
When using collection export:
`{"foo":"testing (bar)"}` becomes `{"foo":"testing (bar }"}` using either regular or JSON array export.

**Comment by Windslash** [comment]
When exporting a `Date` field, the JSON of Date part will end with a `}`.
```
{"_id":{ "$oid": "57dbdfdd638867c7927c51ef" },"date":"2016-09-16T12:04:45.196Z" }}
```

**Comment by dozoisch** [comment]
I have the same issue as @Windslash. Any exported date messes up the JSON and makes it unparseable.

**Comment by wulfsolter** [comment]
PRs welcome. Issue resides in https://github.com/mongo-express/mongo-express/blob/master/lib/json.js or bson.js.

**Comment by dozoisch** [comment]
I'll send a PR tomorrow when I get time!

**Comment by ionutandone** [comment]
Why is this closed? The problem still persists for all strings that contain ")".
https://github.com/mongo-express/mongo-express/blob/master/lib/bson.js#L125

**Comment by wulfsolter** [comment]
Got a PR @ionutandone ?

**Comment by dozoisch** [comment]
I'll try to change that part completely so it matches the full regex expression and not just closing parenthesis when I get time this week.

**Comment by dozoisch** [comment]
closed with #306 a while ago
